osier/technology.py

```diff
@@ -214,11 +252,16 @@ def __init__(self,
                  fuel_type=None,
                  capacity=0.0,
                  capacity_factor=1.0,
+                 co2_rate=0.0,
+                 land_intensity=0.0,
                  efficiency=1.0,
                  lifetime=25.0,
                  default_power_units=MW,
                  default_time_units=hr,
-                 default_energy_units=None) -> None:
+                 default_energy_units=None,
```

Comment:
Any reason you don't define a default energy unit?

Reply:
Yes, because it should be consistent with the power and time units. It is a derived unit.

Reply:
As in in your code it's a derived unit? Because in physics power is the derived unit? And if you code treats energy as a derived unit, can't you just use a default of `default_power_units*default_time_units`?

Reply:
Yes, in this code energy is the derived unit because defining the energy output of a technology doesn’t make much sense. 
Indeed that’s what I do! However, I use a setter method for all of the unit attributes because they undergo unit validation.

Reply:
Alright. That makes sense.